Misc/NEWS.d/next/Library/2019-01-14-11-53-10.bpo-34294.3JFdg2.rst

```diff
@@ -0,0 +1 @@
+re module, fix wrong capturing groups in rare cases. :func:`re.search`, :func:`re.findall`, :func:`re.sub` and other functions that scan through string looking for a match, should reset capturing groups between two match attempts.
```

Comment:
Please wrap the long line at 80 characters and add "Patch by *yourname*."

Reply:
Thanks for your review, and sorry for my unskilled.